packages/wrangler/src/experimental-assets.ts

```diff
@@ -133,6 +133,14 @@ export const syncExperimentalAssets = async (
 					const decodedFilePath = decodeFilePath(manifestEntry[0], path.sep);
 					const absFilePath = path.join(assetDirectory, decodedFilePath);
 
+					let contentType = getType(absFilePath) || "application/octet-stream";
+					if (
+						contentType.startsWith("text/") &&
+						!contentType.includes("charset")
+					) {
+						contentType = `${contentType}; charset=utf-8`;
+					}
```

Comment:
Could share this with [the same code in the index file](https://github.com/cloudflare/workers-sdk/pull/6631/commits/f9c71359e848791d613ff703949130ec4afaccb5#diff-7ff83250c22d1ffda26334752afe5b193eb685e491960f8ab94adbfe5d608dadR290-R296)?

Reply:
indeed. This was originally extracted in a separate fn [here](https://github.com/cloudflare/workers-sdk/pull/6631/commits/f9c71359e848791d613ff703949130ec4afaccb5#diff-5a63ebaa83d43a8f47c269b3a4a28584506d7c439860b372b111cc3ecfa6dccaL10-L21). I'm assuming that was removed because we no longer need to merge those headers? However, if we need to re-use that code, might make sense to export this bit again